src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react'
```

Comment:
The cart page pull together the Actions Required list and the Available For Download files table.  It also handles the case when these are empty (based on the Download List v2 statistics), so those components are not loaded if they are empty.  This makes Clearing the list easy too (since it refetches the stats, and will not show the child components if they are empty).

Reply:
Super cool to use the stats to determine to render the sub-components without having to do cache invalidation. I'm wondering what happens when

1. User has items in the download list (cache is populated in AvailableForDownloadTable with some nonzero staleTime)
2. User clears the download list (unmounting AvailableForDownloadTable)
3. User adds an item to the download list before the staleTime expires

If you used a persistent cache, I suspect a new AvailableForDownloadTable would retrieve the old download list.

Right now, I actually think it works because you don't use a persistent cache; you create a new QueryClient every time you render an AvailableForDownloadTable. At first glance, it also seems like none of the hooks you're using would benefit much from cache sharing, so you can ignore [this comment](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/1086/files#r655661980). So I have no problem with this, but I think we should capture this complexity in a comment so we don't mess with this later and wonder why the cache no longer magically works.